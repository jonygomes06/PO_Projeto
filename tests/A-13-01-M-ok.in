1
faltoso
7
1
2
2
1
3
3
0
3
4
20
7
2
2
2
n
2
3
3
n
0
5
3
5
2
3
5
3
3
0
0
