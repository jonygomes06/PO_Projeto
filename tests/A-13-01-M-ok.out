Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Ficheiro a abrir: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 2 deve ser devolvida até ao dia 23.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 3 deve ser devolvida até ao dia 36.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Data actual: 21
Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Introduza número de dias a avançar: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: O utente 2 deve pagar uma multa de EUR 90.
O utente deseja pagar multa (s/n)? Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Introduza o número de utente: Introduza o número da obra: O utente 3 deve pagar uma multa de EUR 25.
O utente deseja pagar multa (s/n)? Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 1 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
2 - Darth Vader - darth@dark.force.coAm - FALTOSO - SUSPENSO - EUR 90
3 - Luke Skywalker - last.jedi@universe.com - NORMAL - SUSPENSO - EUR 25
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Introduza o número de utente: Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 1 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
2 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
3 - Luke Skywalker - last.jedi@universe.com - NORMAL - SUSPENSO - EUR 25
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Introduza o número de utente: Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 1 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
2 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
3 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 